README.md

```diff
@@ -48,19 +48,6 @@ CtClass l = Launcher.parseClass("class A { void m() { System.out.println(\"yeah\
 
 Some examples about the usage of Spoon can be found on [spoon-examples](https://github.com/SpoonLabs/spoon-examples).
 
-:warning: We introduced in Spoon 7.1.0 an external dependency that is not available on Maven Central. 
```

Comment:
This warning is still true for 7.1.0. Let's wait the release before removing it.

Reply:
I really don't want to be annoying with this, so if you want I'll restore it. But, to my knowledge, the part about the need to insert the additional repository for user of spoon, is simply not true.

Reply:
Ahah still a bit sensitive on that one I guess ;)
for me it's really more to avoid having user complaints in case they got problem because of that. I never manage to reproduce the error neither, so.... + I really want to release quickly your new module, to avoid this dependency issue. 

Now it's up to you

Reply:
I have restored it with small modifications. If you don't like them, I'll just restore your version.

Reply:
@surli WDYT?

Reply:
IMO the message for 7.2.0 is useless, since the warning will be removed when Spoon 7.2.0 will be released.

Reply:
I agree. I'll remove it then.